Customization/T240/CodeSnippets/Step1.1.2/RSSVWorkOrderEntry.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections;
+using PX.Data;
+using PX.Data.BQL.Fluent;
+using PX.Objects.IN;
+using PX.Data.BQL;
+using PX.Data.WorkflowAPI;
+using PX.Objects.AR;
+using PX.Objects.SO;
+using PX.Objects.AR;

```

Comment:
You have two directives with PX.Objects.AR and System.Collections. This generates a warning in VS.